soci/soci_index_test.go

```diff
@@ -142,10 +142,10 @@ func TestBuildSociIndexNotLayer(t *testing.T) {
 				MediaType: tc.mediaType,
 				Digest:    "layerdigest",
 			}
-			cfg := &buildConfig{}
 			spanSize := int64(65535)
 			blobStore := memory.New()
-			_, err := buildSociLayer(ctx, cs, desc, spanSize, blobStore, cfg)
+			builder, _ := NewIndexBuilder(cs, blobStore, WithSpanSize(spanSize), WithMinLayerSize(0))
```

Comment:
We only need one ``IndexBuilder`` for all test cases, so ``NewIndexBuilder`` should be moved out of this ``for`` loop.